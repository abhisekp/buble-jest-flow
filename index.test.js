const transformer = require('./index');

test('buble jest remove flow types', () => {
  expect(transformer.process('(num: number): number => 0', { all: true })).toMatchSnapshot();
});
