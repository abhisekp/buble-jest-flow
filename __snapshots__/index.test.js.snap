// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`buble jest remove flow types 1`] = `
"\\"use strict\\";

!function(num) { return 0; };"
`;
